fenix/app/src/test/java/org/mozilla/fenix/shopping/fake/FakeReviewQualityCheckVendorsService.kt

```diff
@@ -15,4 +15,6 @@ class FakeReviewQualityCheckVendorsService(
     ),
 ) : ReviewQualityCheckVendorsService {
     override fun productVendors(): List<ProductVendor> = productVendors
+
+    override fun getProductVendor(): ProductVendor = productVendors().first()
```

Comment:
Yeah I don't think we need this function in the API as we are duplicating the logic in real and fake, the tests should pass in the `productVendors` when creating `FakeReviewQualityCheckVendorsService` and then assert expected to be the first item in the passed list.